documentation/asciidoc/microcontrollers/microcontroller_docs.adoc

```diff
@@ -24,4 +24,9 @@ https://datasheets.raspberrypi.org/pico/raspberry-pi-pico-c-sdk.pdf[Raspberry Pi
 
 https://datasheets.raspberrypi.org/pico/raspberry-pi-pico-python-sdk.pdf[Raspberry Pi Pico Python SDK]:: A MicroPython environment for RP2040 microcontrollers
 
-The API level Doxygen documentation for the Raspberry Pi Pico C/C++ SDK is also available https://rptl.io/pico-doxygen[as a micro-site].
\ No newline at end of file
+The API level Doxygen documentation for the Raspberry Pi Pico C/C++ SDK is also available https://rptl.io/pico-doxygen[as a micro-site].
```

Comment:
Should API-level be hyphenated?

Reply:
Not hyphenated on the RP2040 Landing Page where this text came from and that's been through multiple copyedits by the comms team, so I'm guessing not. I'll ask Helen and Liz though.